.github/workflows/build-bootc.yaml

```diff
@@ -5,11 +5,17 @@ on:
     - cron: '0 */12 * * *'
 
 env:
-  QUAY_ORG: quay.io/flightctl
+  REGISTRY: quay.io
+  REGISTRY_USER: ${{ github.actor }}
 
 jobs:
   build-and-push-centos:
```

Comment:
`build-and-push-centos` and `build-and-push-fedora` are almost identical can you use a [github actions matrix](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) to simplify the workflow file?